sharing-wp-cli-packages.md

```diff
@@ -0,0 +1,49 @@
+# Sharing WP-CLI Packages
+
+By default, WP-CLI places [installed packages](https://developer.wordpress.org/cli/commands/package/) in `~/.wp-cli/packages/`, a hidden subdirectory for the user’s home directory.
+
+Because the home directory is different for each user, this naturally means each system user will have a separate directory of installed packages. If you have multiple active shell users on a server, and want to share installed WP-CLI packages between them, there are a couple of supported ways to do this.
+
+## `WP_CLI_PACKAGES_DIR` environment variable
+
+To override the directory WP-CLI uses for installed packages, provide a `WP_CLI_PACKAGES_DIR` environment variable. If you wish for multiple users to share the same packages directory, you can simply provide the same `WP_CLI_PACKAGES_DIR` environment variable for each user.
+
+    vim /etc/environment
+    export WP_CLI_PACKAGES_DIR=/usr/local/bin/wp-cli-packages
```

Comment:
Using `/usr/local/bin/` to put the packages folder into does not make sense. That folder should only contain executable files.

How about using `/usr/local/lib/wp-cli-packages` as an example?

Reply:
bfeea0e